### PR TITLE
Add mobile option to image_stylization

### PR DESCRIPTION
**Pull request by khanhlvg**
* Added an alpha multiplier to image_stylization to allow slimming down the model to run faster on mobile
* Added a script and instruction on how to convert the model to TF Lite
* Adapted the existing scripts to run on both Python 2 and 3
* Showed stylization sample on TensorBoard during training to allow users to better understand the training progress.

### CODE REVIEW

**Comment by khanhlvg** [comment]
For the record: This is the original image (Free for commercial use, No attribution required) used to generate the benchmark image.
https://pixabay.com/photos/men-women-apparel-couple-people-2425121/

File: magenta/models/image_stylization/image_stylization_finetune.py [hunk 1]
```diff
@@ -126,18 +128,13 @@ def main(unused_argv=None):
                     if 'InstanceNorm' not in var.name]
 
       # Function to restore VGG16 parameters.
-      # TODO(iansimon): This is ugly, but assign_from_checkpoint_fn doesn't
-      # exist yet.
-      saver_vgg = tf.train.Saver(slim.get_variables('vgg_16'))
-      def init_fn_vgg(session):
-        saver_vgg.restore(session, vgg.checkpoint_file())
+      init_fn_vgg = slim.assign_from_checkpoint_fn(vgg.checkpoint_file(),
+                                     slim.get_variables('vgg_16'))
```
Comment (vdumoulin) [inline magenta/models/image_stylization/image_stylization_finetune.py]: Should this line be aligned with the opening parenthesis?

File: magenta/models/image_stylization/image_stylization_finetune.py [hunk 2]
```diff
@@ -126,18 +128,13 @@ def main(unused_argv=None):
...
+
 
       # Function to restore N-styles parameters.
-      # TODO(iansimon): This is ugly, but assign_from_checkpoint_fn doesn't
-      # exist yet.
-      saver_n_styles = tf.train.Saver(other_vars)
-      def init_fn_n_styles(session):
-        saver_n_styles.restore(session, os.path.expanduser(FLAGS.checkpoint))
+      init_fn_n_styles = slim.assign_from_checkpoint_fn(os.path.expanduser(FLAGS.checkpoint),
+                                     other_vars)
```
Comment (vdumoulin) [inline magenta/models/image_stylization/image_stylization_finetune.py]: Should this line be aligned with the opening parenthesis?

**Comment by khanhlvg** [comment]
Thanks Vincent. I've fixed the incorrect indent.

**Pull request merged**